arm-recoveryservices/2016-06-01/swagger/recoveryservices.json

```diff
@@ -0,0 +1,342 @@
+            "schema": {
+              "$ref": "#/definitions/Vault"
+            }
+          }
+        },
+        "deprecated": false
+      },
+      "delete": {
+        "tags": [
+          "Vaults"
+        ],
+        "description": "Deletes a vault.",
+        "operationId": "Vaults_Delete",
+        "produces": [
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/VaultName"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "name": "isGlobalRPContext",
+            "in": "query",
+            "required": false,
+            "type": "boolean"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          }
+        },
+        "deprecated": false
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/vaults": {
+      "get": {
+        "tags": [
+          "Vaults"
+        ],
+        "summary": "Retrieve a list of Vaults.",
+        "operationId": "Vaults_ListByResourceGroup",
+        "produces": [
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/VaultList"
+            }
+          }
+        },
+        "deprecated": false,
+		"x-ms-pageable": {
+          "nextLinkName": "null"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Object": {
+      "description": "Base of all objects.",
+      "required": [],
+      "type": "object",
+      "properties": {}
+    },
+    "VaultList": {
+      "description": "The response model for Vault.",
+      "required": [],
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Vault"
+          }
+        }
+      }
+    },
+    "Vault": {
+      "description": "Resource information, as returned by the resource provider.",
+      "required": [
+        "location"
+      ],
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "properties": {
+        "etag": {
+          "description": "ETag of the Vault.",
+          "type": "string",
+          "readOnly": true
+        },
+        "properties": {
+          "$ref": "#/definitions/VaultProperties"
```

Comment:
I've checked that the service side call fails when I apply this extension. Thus, I'm omitting this one.